crates/bevy_sprite/src/entity.rs

```diff
@@ -63,10 +63,14 @@ impl Default for SpriteComponents {
     }
 }
 
+/// A Bundle for components related to sprite sheets (a single atlas containing
```

Comment:
I would be a bit more explicit here. Ex:

`A Bundle of components used to draw a single sprite from a sprite sheet (also known as a TextureAtlas).`

As an aside: we should probably just use one name (Sprite Sheet vs Texture Atlas). But thats definitely for a different time :smile:

Reply:
Yeah, I was wondering which name would be clearer to someone reading the docs. I personally prefer sprite, but you're right that we can figure that out at a different time